GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -196,5 +243,31 @@ private async void GetSubject()
             SubjectImageSource = CurrentSubject.GetSubjectLocation();
             Subjects.RemoveAt(0);
         }
+
+        private void LevelUp()
+        {
+            if (ClassificationsThisSession > 25)
```

Comment:
Just curious why you have this guard? Seems fairly pointless if the switch will just return "one"